src/node.h

```diff
@@ -349,8 +349,12 @@ inline std::unique_ptr<InitializationResult> InitializeOncePerProcess(
 }
 
 enum OptionEnvvarSettings {
-  kAllowedInEnvironment,
-  kDisallowedInEnvironment
+  kAllowedInNodeOptions = 0,
```

Comment:
We can always just add a comment here that clarifies what this is actually for ;)